src/seedu/addressbook/parser/Parser.java

```diff
@@ -94,6 +84,9 @@ public Command parseCommand(String userInput) {
         case ViewAllCommand.COMMAND_WORD:
             return prepareViewAll(arguments);
 
+        case SortCommand.COMMAND_WORD:
+            return prepareSort();
```

Comment:
I don't think you need to prepareSort() as there is nothing to prepare. You could just return new SortCommand()